pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
```

Comment:
We tend to prefer descriptions that actually describe what the command *does*. In this case, for example, "Connect to a remote server" would probably work.